calcom/cal.com
# fix: Make routing-form a preinstalled app

## What does this PR do?

Fixes #10372
Fixes #9351

Routing Forms is a Default app now, globally installed.
![image](https://github.com/calcom/cal.com/assets/1780212/5864756b-eb1e-4553-8bf9-20b1a204740c)

It will remain accessible at` /apps/routing-forms `but will be referred to as `/routing-forms` throughout the app.

[Demo Loom](https://www.loom.com/share/6d64832e8ff242a78b4aa338fa0be3ee)

## Type of change
- Improvement

## How should this be tested?
- See the loom
- You simply need to create a new account and you would see Routing Forms App preinstalled.
## Mandatory Tasks


## Checklist
- I haven't checked if my changes generate no new warnings
